tests/test_values.py

```diff
@@ -129,13 +129,29 @@ def test_boolean_values_assign_false_to_another_booleanvalue(self):
         self.assertFalse(value1.setup('TEST1'))
         self.assertFalse(value2.setup('TEST2'))
 
+    def test_direct_integer_value(self):
+        error_values = ('1.5', float(1.5))
+        values = ('1', 1)
+        for each in error_values:
+            self.assertRaises(ValueError, IntegerValue, each)
+        for each in values:
+            self.assertEqual(IntegerValue(each), each)
```

Comment:
This is the default behavior. However, `self.assertEqual(IntegerValue(each), int(each))` is more suited.